addons/html_editor/static/src/core/history_plugin.js

```diff
@@ -752,7 +752,8 @@ export class HistoryPlugin extends Plugin {
      */
     makeSavePoint() {
         this.handleObserverRecords();
-        const currentStepMutationsUntilLength = this.currentStep.mutations.length;
+        const currentMutations = [...this.currentStep.mutations];
```

Comment:
i just benchmarked, `list.slice()` vs `[...list]`. well, slice is more than 30 times faster